pages/services/spark/2.3.1-2.2.1-2/release-notes/index.md

```diff
@@ -44,10 +45,11 @@ render: mustache
 - Changed the default user to root. (Breaking change)
 
 ### Bug fixes
-- First delegation token renewal time is not 75% of renewal time. (SPARK-22583)
-- Fixed supervise mode with checkpointing. (SPARK-22145)
+- First delegation token renewal time is not 75% of renewal time. [SPARK-22583](https://issues.apache.org/jira/browse/SPARK-22583)
+- Fixed supervise mode with checkpointing. [SPARK-22145](https://issues.apache.org/jira/browse/SPARK-22145)
 - Added support for older SPARK_MESOS_KRB5_CONF_BASE64 environment variable.
 - The spark CLI has "shortcut" command-line args, that are translated into spark.config=setting configurations downstream (such as spark.executor.memory). Fixed a bug where a user sets the configuration directly and is overwritten with the default value for the shortcut argument.
+- Fixed Spark SQL failure to create a Hive table when running in a Apache Sentry-secured Environment. [SPARK-24338](https://issues.apache.org/jira/browse/SPARK-24338)
```

Comment:
This is definitely not a bug fix in package `2.3.0-2.2.1-2` as this is something we are currently working on and plan to release in `2.4.0-2.2.1-3` (and maybe even after as we are not certain it will get into this release)

Reply:
I took this line back out.